scripts/main/album.js

```diff
@@ -298,6 +302,100 @@ album.add = function (IDs = null, callback = null) {
 };
 
 
+album.addByTags = function (IDs = null, callback = null) {
+
+	const action = function (data) {
+
+		basicModal.close();
+
+		let params = {
+			title: data.title,
+			tags: data.tags,
+			parent_id: 0
+		};
+
+		api.post('Album::addByTags', params, function (_data) {
+			const isNumber = (n) => (!isNaN(parseInt(n, 10)) && isFinite(n));
+			if (_data !== false && isNumber(_data)) {
+				if (IDs != null && callback != null) {
+					callback(IDs, _data, false); // we do not confirm
```

Comment:
What is that callback for? I don't think you use it?

Reply:
removed callback and IDs parameter too